lib/pages/labels/label_db.dart

```diff
@@ -47,7 +47,7 @@ class LabelDB {
     return labels;
   }
 
-  Future deleteLabel(int labelId) async {
+  Future deleteLabel(int? labelId) async {
```

Comment:
Is this change for deleting a label with `labelId` `null`? Can this happen?

Reply:
yes

Reply:
int? can take  null value also

Reply:
Why would int will be null for deleting the project?

Reply:
sir when we are building list of labels it may give us value null, and whenever i was using int it was showing me error that int cant be assigned to type int?

Reply:
First of all. You don't need to call me sir.
You can mark `deleteLabel` int as non-nullable and before calling this method check if the argument passed in `deleteLabel` is not null. Something like

```dart
if (lable.id! = null) {
  labelDb.deleteLabel(label.id);
}
```

Reply:
ok
i will go through it.

Reply:
it dose not  worked for me, when I tried to remove int? it was showing error: The argument type 'int?' can't be assigned to the parameter type 'int'. How can  I deal with it